kratos/includes/linear_master_slave_constraint.h

```diff
@@ -135,132 +211,248 @@ class LinearMasterSlaveConstraint :  public MasterSlaveConstraint
 
     /// Copy Constructor
     LinearMasterSlaveConstraint(const LinearMasterSlaveConstraint& rOther)
+        : BaseType(rOther),
+          mSlaveDofsVector(rOther.mSlaveDofsVector),
+          mMasterDofsVector(rOther.mMasterDofsVector),
+          mRelationMatrix(rOther.mRelationMatrix),
+          mConstantVector(rOther.mConstantVector)
     {
-        this->SetId(rOther.Id());
-        // this->Flags = rOther.Flags;
-        this->mSlaveDofsVector = rOther.mSlaveDofsVector;
-        this->mMasterDofsVector = rOther.mMasterDofsVector;
-        this->mRelationMatrix = rOther.mRelationMatrix;
-        this->mConstantVector = rOther.mConstantVector;
     }
 
     /// Assignment operator
     LinearMasterSlaveConstraint& operator=(const LinearMasterSlaveConstraint& rOther)
     {
-        this->SetId(rOther.Id());
-        // this->Flags = rOther.Flags;
-        this->mSlaveDofsVector = rOther.mSlaveDofsVector;
-        this->mMasterDofsVector = rOther.mMasterDofsVector;
-        this->mRelationMatrix = rOther.mRelationMatrix;
-        this->mConstantVector = rOther.mConstantVector;
-
+        BaseType::operator=( rOther );
+        mSlaveDofsVector = mSlaveDofsVector;
```

Comment:
this should be 
` mSlaveDofsVector = rOther.mSlaveDofsVector;` or ?

also in the following lines